v4/export/config.go

```diff
@@ -53,7 +53,7 @@ type Config struct {
 	FileType          string
 	EscapeBackslash   bool
 	DumpEmptyDatabase bool
-	TiDBMemQuotaQuery uint64
+	SessionParams     map[string]string
```

Comment:
perhaps this should be

```suggestion
	SessionParams     map[string]interface{}
```

so that we could assign a number to it

Reply:
But we still have to transfer it to string for dsn 🤔